python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -898,6 +923,37 @@ def _fast_slow_function_call(func: Callable, /, *args, **kwargs) -> Any:
                 # try slow path
                 raise Exception()
             fast = True
+            if cudf_pandas_debugging:
+                try:
+                    with nvtx.annotate(
+                        "EXECUTE_SLOW_DEBUG",
+                        color=_CUDF_PANDAS_NVTX_COLORS["EXECUTE_SLOW"],
+                        domain="cudf_pandas",
+                    ):
+                        slow_args, slow_kwargs = (
+                            _slow_arg(args),
+                            _slow_arg(kwargs),
+                        )
+                        with disable_module_accelerator():
+                            slow_result = func(*slow_args, **slow_kwargs)
+                except Exception as e:
+                    warnings.warn(
+                        "The result from pandas could not be computed. "
+                        f"The exception was {e}."
+                    )
```

Comment:
Should we make these warnings use a custom warning class, like this? It would be helpful if we need to filter these out if treating warnings as errors. I can go either way on this, we could always add that feature later.
```python
class CudfPandasDebugWarning(UserWarning):
    pass
```
```suggestion
                    warnings.warn(
                        "The result from pandas could not be computed. "
                        f"The exception was {e}.", CudfPandasDebugWarning
                    )
```

Reply:
Okay, Ill add those changes in the next PR that adds the debug fallback environment variable